piqueserver/scripts/savemap.py

```diff
@@ -18,32 +19,52 @@
 from twisted.logger  import Logger
 from pyspades.vxl import VXLData
 from piqueserver.config import config
+from piqueserver.commands import command
 
+savemap_config = config.section('savemap')
 log = Logger()
 
-savemap_config = config.section('savemap')
-LOAD_SAVED_MAP_OPTION = savemap_config.option('load_saved_map', False)
+@command('save', admin_only=True)
+def save(connection):
+    connection.protocol.save_map()
+
+@command('toggle_load_saved_map', admin_only=True)
+def toggle_load_saved_map(connection):
```

Comment:
I'm not convinced this is something you'd want to turn on and off regularly enough to justify an in-game command. Especially since the server has already started and loaded a map at this point.